.github/workflows/project-automation.yml

```diff
@@ -0,0 +1,73 @@
+name: Project automation
+
+on:
+  pull_request:
+  issues:
+  issue_comment:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Dreamcodeio/pr-has-label-action@v1.2
+        id: checkLabel
+        with:
+          label: stable backport
+
+      - name: Remove from 'Release Next' project
```

Comment:
It should probably be in both projects, since it's merged in the next release, but then backported to the current one.